ui-kit/src/main/kotlin/org/p2p/uikit/atoms/skeleton/UiKitSkeletonLineViewModel.kt

```diff
@@ -0,0 +1,7 @@
+package org.p2p.uikit.atoms.skeleton
+
+data class UiKitSkeletonLineViewModel(
```

Comment:
our first `ViewModel` in the project @kabramovp2p  😄 

UiKitSkeletonLineModel / UiKitSkeletonLineItem would be ok